packages/alfa-http/test/request.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Request } from "../src/request";
+
+test(".from() returns Err on invalid URL", (t) => {
+  t.deepEqual(
+    Request.from({
+      url: "foo",
+      body: "",
+      headers: [],
+      method: "GET",
+    }).toJSON(),
+    {
+      type: "err",
+      error: "Invalid URL",
```

Comment:
Annoyingly, the behavior is different between Node 14 and 16. In 14 the error message is "Invalid URL: foo" and in 16 it's just "Invalid URL". I'll need to make the tests able to cope with this.